### DISCUSSION TITLE
function of metaMDS points attribute: internalscaling

### DISCUSSION THREAD

**Discussion by khaosn** [vegandevs/vegan#555] [Q&A]
I tried to redo an MDS analysis, originally made in Primer6, in vegan and obtained approximately the same results, except that the results differed by a factor. I eventually found this factor to be equivalent to the internalscaling attribute of the points of the metaMDS object. What purpose does the internascaling factor have in metaMDS? Thanks.

**Comment by jarioksa** [comment]
The scaling is discussed in the help page of `metaMDS` (see `help(metaMDS)`, point 5 in section Details).

In general, the scaling of scores is arbitrary in NMDS and absolute scores or absolute distances between points in ordination should not be compared. The only thing that matters is the configuration. However, `vegan::metaMDS` tries to make scores such that, depending on the options, either (1) community similarity halves at one unit of ordination scores, or (2) ordination scores have the same range as the input dissimilarities. However, any other scaling can be used and are equally justified.
